transactionoutbox-spring/src/test/java/com/gruelbox/transactionoutbox/acceptance/EventuallyConsistentControllerTest.java

```diff
@@ -30,7 +30,7 @@ void setUp() throws Exception {
   void testCheck() throws Exception {
     ResponseEntity<String> response =
         template.getForEntity(base.toString() + "/createCustomer", String.class);
-    assertThat("Done").isEqualTo(response.getBody());
+    assertThat("Done".equals(response.getBody()));
```

Comment:
Pretty sure this is a regression. The old version wasn't actually checking anything.

Reply:
tnx, fixed